src/engine/guidance/lane_processing.cpp

```diff
@@ -96,7 +96,7 @@ std::vector<RouteStep> anticipateLaneChange(std::vector<RouteStep> steps,
 
             // only prevent use lanes due to making all turns. don't make turns during curvy
             // segments
-            if (previous_inst.type == TurnType::UseLane)
+            if (previous_inst.type == TurnType::Suppressed)
```

Comment:
yes, it will trigger more often, but that is fine. Time_to_constrained will only increase, if we have suppressed instructions here. After all a suppressed turn is not influencing our ability to make the turn, right? Just as use-lane. So we only increase our chance to `can_make_all_turns`.